cdci_data_analysis/analysis/parameters.py

```diff
@@ -461,7 +461,12 @@ def reprJSONifiable(self):
         if restrictions:
             reprjson[0]['restrictions'] = restrictions
         if getattr(self, 'owl_uris', None):
-            reprjson[0]['owl_uri'] = self.owl_uris
+            if isinstance(self.owl_uris, str):
+                reprjson[0]['owl_uri'] = [ self.owl_uris ]
```

Comment:
in case it's a string, how is it going to be formatted ?as a comma-separated list of uris?

perhaps it's safe to `split(',')` it?

Reply:
In this case, it's just one URI. This is not intentional, but we already have a situation where `owl_uris` attribute is either tuple or string. 
E.g. here https://github.com/oda-hub/dispatcher-app/blob/420e273c92261c3d112a1db9e2b599b84768247f/cdci_data_analysis/analysis/parameters.py#L559 it's meant to be a tuple, but it's actually a string, because of a missing comma. But everything works already anyway, and moreover, most parameters will have a single URI.

Here I just normalize it to always be a list in the metadata to simplify further checks.